mix.exs

```diff
@@ -67,13 +61,12 @@ defmodule Expo.MixProject do
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:ex_doc, ">= 0.0.0", only: [:dev], runtime: false},
```

Comment:
@whatyouhide Why are we removing the runtime part?

Reply:
@maennchen do we not want to start these applications in dev? If so, why?

Reply:
@whatyouhide all of thise have specific commands to run them. I don’t think they should be running in any other case.

Reply:
Alright, reverted this change!